projects/super_res/config_mod_flow.py

```diff
@@ -0,0 +1,43 @@
+from ml_collections import config_dict
```

Comment:
Can we use a different name than "mod"? Is this the config for the 2D flow?

Reply:
Sure